website_img_dimension/static/src/js/website_img_dimension.tooltip.js

```diff
@@ -0,0 +1,40 @@
+odoo.define('website_img_dimension.tooltip', function(require) {
+    "use strict";
+    var Class = require('web.Class');
+    var web_editor_base = require('web_editor.base');
+
+    var TooltipManager = Class.extend({
+        init: function() {
+            var self = this;
+            web_editor_base.ready().done(function() {
+                $('#wrap').on('mouseenter', 'img', function() {
+                    self.show_tooltip($(this));
+                });
+
+                $('#wrap').on('mouseleave', 'img', function() {
+                    self.hide_tooltip($(this));
+                });
+            });
+        },
+        get_title: function($img) {
+            return $img.width() + ' x ' + $img.height() + ' (' + $img[0].naturalWidth + ' x ' + $img[0].naturalHeight + ')';
+        },
+        show_tooltip: function($img) {
+            $img.tooltip({
+                title: this.get_title($img),
+                trigger: 'manual',
+                container: 'body'
+            }).tooltip('show');
+        },
+        hide_tooltip: function($img) {
+            $img.tooltip('destroy');
+        }
+    });
+
+    var tooltip_manager = new TooltipManager();
+
+    return {
```

Comment:
Why returning both an instance and the Class itself?

Reply:
I return the class so other modules can extend the TooltipManager, and I return the instance so other modules can invoke the functions or access variables of it. As TooltipManager is normally only created once like web.WebClient / web.web_client